BHoM_Engine/Query/PropertyValue.cs

```diff
@@ -59,45 +62,57 @@ public static object PropertyValue(this object obj, string propName)
 
             System.Reflection.PropertyInfo prop = obj.GetType().GetProperty(propName);
 
+            object result = null;
             if (prop != null)
-                return prop.GetValue(obj);
-            else 
-                return GetValue(obj as dynamic, propName);
+                result = prop.GetValue(obj);
+            else
+                result = GetValue(obj as dynamic, propName);
+
+            if (result is IQuantity)
+                result = ((IQuantity)result).Value;
```

Comment:
This does not handle collections, leading it to crash (cast exception) in the UI for a Property of type List<_someQuantity_>.

Something like the below resolve my test case, but not sure if even more might be required. The below is also changing the return type (if the Property was any other collection type than a List), which might or might not be a problem.

```suggestion
            if (result is IQuantity)
                result = ((IQuantity)result).Value;
            else if(result is IEnumerable<IQuantity>)
                result = (result as IEnumerable<IQuantity>).Select(x => x.Value).ToList();
```

Reply:
AFter discussion on call. THis generally does not make sense to have a list of quantities, or at least not jsut to return them as doubles, as that will be information loss, as the quantity can vary.

Happy for this comment to be disregarded.